src/category_theory/groupoid/basic.lean

```diff
@@ -0,0 +1,46 @@
+/-
+Copyright (c) 2022 Rémi Bottinelli. All rights reserved.
+Released under Apache 2.0 license as described in the file LICENSE.
+Authors: Rémi Bottinelli
+-/
+import category_theory.groupoid
+import category_theory.is_connected
+
+/-!
+This file defines a few basic properties of groupoids.
+-/
+
+namespace category_theory
+
+namespace groupoid
+
+variables (C : Type*) [groupoid C]
+
+section graph_like
+
+/-- A groupoid is graph-like if it has no parallel arrows. -/
+def is_graph_like := ∀ (c d : C), subsingleton (c ⟶ d)
```

Comment:
Is this standard terminology? I usually see "thin" (as in thin category) or "setoid" (c.f. [stacks#04S9](https://stacks.math.columbia.edu/tag/04S9)) but I might have different backgrounds.

Reply:
It's not really standard. I like "thin" better, but isn't it somewhat confusing in addition to "wide" ?

Reply:
Wikipedia and nlab says that it is also called `posetal`, though I don't really like this name. I don't think "thin" is that confusing since wide and thin are both standard terminology and they are technically not antonyms.